### ISSUE TITLE
Missplaced comments before moves

### ISSUE THREAD

**Issue by juanparra64** [yo35/rpb-chessboard#176]
Hello.

I have this simple PGN test:
```[pgn_compat piece_symbols=figurines navigation_board=frame]
[Event ""]
[Date "2020.09.21"]
[White "RPB Chessboard Test"]
[Result "*"]
1. e4 e5 2. Nf3 Nc6 3. Bb5 {White play the Spanish.} ({Today is very popular
the Italian begginning with} 3. Bc4) {Now, Black play the Morphy Defense with}
3... a6 4. Ba4 ({The Exchange Variation with} 4. Bxc6 {is less agressive.})
4... Nf6 5. O-O Be7 6. Re1 b5 7. Bb3 d6 8. c3 O-O 9. h3 *
[/pgn_compat]
```
The comment “Now, Black play the Morphy Defense with” must be shown just before the 3…a6 move. But it is placed just after the 3.Bb5 move! That is:
```PB Chessboard Test
21 de septiembre de 2020
1.e4 e5 2.Nf3 Nc6 3.Bb5 Now, Black play the Morphy Defense with (Today is very popular the Italian begginning with 3.Bc4) 
3…a6 4.Ba4 (The Exchange Variation with 4.Bxc6 is less agressive.) 4…Nf6 5.O-O Be7 6.Re1 b5 7.Bb3 d6 8.c3 O-O 9.h3
```
What’s wrong?

Thanks for your support and best regards.

**Comment by yo35** [comment]
Hi,
The plugin associate each text comment to the preceding move within the same variation. In your example:
```
... 3.Bb5 {White plays the Spanish.} (... a sub-variation...) {Now, Black ...} 3...a6 ...
           ^                              ^                    ^
           |                              |                    Associated to the preceding move too, that is 3.Bb5.
           |                              |                    It replaces "White plays the Spanish". 
           |                              Sub-variation, not part of the main one
           This comment is associated to the preceding move within the same variation, that is 3.Bb5
```
What is obtained at the end corresponds to what is expected for a comment associated to `3.Bb5`.

I suggest you modified your game in the following way:
```
... 3.Bb5 {White plays the Spanish.} (... the sub-variation ...) 3...a6 {Now, Black plays the Morphy Defense} 4.Ba4 ...
```
... which should a more readable result.

**Comment by juanparra64** [comment]
Hi, @yo35

You are right. I have the game in ChessBase as I described, with the comment before 3...a6. I exported the game to PGN, then I opened the PGN in ChessBase again..., and the comment is missplaced after 3.Bb5 too! So, I notice that it is not a RPB issue, but a PGN problem.
![01 Original test in ChessBase](https://user-images.githubusercontent.com/71660748/93851747-fe5a5d00-fcb0-11ea-9f9f-df7e2c3017bb.png)
![02 Exported PGN opened in ChessBase](https://user-images.githubusercontent.com/71660748/93851752-fef2f380-fcb0-11ea-8226-52a61745bd82.png)

By the way, I hate the PGN format... This was only a test to show you, but I realised the issue with ChessBase games from books, where the author wrote a lot of text with the right line breaks and with a lot of comments before moves. Then, when exported to PGN it results in an embarrassed missplaced text without line breaks and with all the comments after each move... I think there is no solution. PGN is very disappointing for me, but is the only way for inserting a game in a webpage... I think that we need an enhanced PGN 2.0 version according to XXI century...

Thanks for your tips, and best regards.
